src/common/conf.cpp

```diff
@@ -3,14 +3,15 @@
 
 #include "conf.hpp"
 
-#include "showmsg.hpp" // ShowError
+#include "showmsg.hpp"  // ShowError
 
-int conf_read_file(config_t *config, const char *config_filename)
-{
+int conf_read_file(config_t *config, const char *config_filename) {
 	config_init(config);
 	if (!config_read_file(config, config_filename)) {
-		ShowError("%s:%d - %s\n", config_error_file(config),
-		          config_error_line(config), config_error_text(config));
+		ShowError("%s:%d - %s\n",
```

Comment:
Do we really want no column limit? Lines get hard to understand at 140+ characters

Reply:
I feel like these totally depend on the output. Sometimes they can be one line but sometimes like @vstumpf said, it can drag on too far horizontally. The big kicker is to see what this would look like in char/login with all the SQL stuff.

Reply:
Usually the only long lines are output formats, like this. Where it is not really problematic.
I do agree that you should not put that much logic into a single line, but this should already be detected during review progress. I tend to comment negative on overloaded short-ifs.

As for the SQL queries I totally agree with that as well, but it should be possible to break the strings into multiple lines, even with that configuration and that would make sense, if we see that clang format is breaking it somewhere.

But for simple format parameter lists I would tend to say that nowdays programmer's screens and editors can display more than 100 characters.

Reply:
Personally I use a split editor, so it's pretty easy to hit the edge of the screen with no limit
![image](https://github.com/rathena/rathena/assets/6844975/55ce99c5-65b6-48af-af3e-c5dfc54c9670)